ai-platform-unified/notebooks/official/model_monitoring/model_monitoring.ipynb

```diff
@@ -35,17 +35,17 @@
         "\n",
```

Comment:
<p>The short link is fairly non-standard. Can you check with Crystal about that.</p><p> </p><p>Also the GCP link seems wrong (hyphen instead of underscore).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/324/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>